modules/claims_api/app/workers/claims_api/poa_updater.rb

```diff
@@ -7,24 +7,28 @@ module ClaimsApi
   class PoaUpdater
     include Sidekiq::Worker
 
-    def perform(power_of_attorney_id)
+    def perform(power_of_attorney_id) # rubocop:disable Metrics/MethodLength
       poa_form = ClaimsApi::PowerOfAttorney.find(power_of_attorney_id)
       service = BGS::Services.new(
         external_uid: poa_form.external_uid,
         external_key: poa_form.external_key
       )
+
+      ssn = poa_form.auth_headers['va_eauth_pnid']
+      file_number = service.people.find_by_ssn(ssn)[:file_nbr] # rubocop:disable Rails/DynamicFindBy
+
       response = service.vet_record.update_birls_record(
-        ssn: poa_form.auth_headers['va_eauth_pnid'],
+        file_number: file_number,
+        ssn: ssn,
         poa_code: poa_form.form_data['serviceOrganization']['poaCode']
       )
 
       if response[:return_code] == 'BMOD0001'
         poa_form.status = ClaimsApi::PowerOfAttorney::UPDATED
+        # Clear out the error message if there were previous failures
+        poa_form.vbms_error_message = nil if poa_form.vbms_error_message.present?
```

Comment:
When debugging some of the jobs that appeared to have initially failed but then succeeded.  This change clears out the error message so that we don't see outdated errors.

In addition to the error message, it looked like a bunch of POAs still at a status of errorred. This makes less sense to me, since we're already changing the status to updated.

My only theory is that by calling [`poa_form.save` instead of `poa_form.save!` below](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/workers/claims_api/poa_updater.rb#L33), it's failing silently instead of raising the issue, but that seems half-baked. Thoughts?

Reply:
The `.save!` is possible but does seem unlikely.
The only thing I can think of is that those that are still errored never made it past this part.  Or maybe they failed at a different point further down then road.

Reply:
I'm trying to understand the reasoning to avoid `.save!` outlined [here](https://stackoverflow.com/questions/5056385/when-to-use-save-vs-save-in-model), but in the meantime, I think we'll see if it continues to be an issue and tackle it accordingly